package.json

```diff
@@ -5,14 +5,19 @@
   "dependencies": {
     "@sentry/browser": "^5.10.0",
     "axios": "^0.19.0",
+<<<<<<< HEAD
+    "material-ui": "^0.20.2",
+=======
     "jquery": "^3.4.1",
+>>>>>>> c3ab826fd1eaec98a219b8838c3d80bd226576c7
```

Comment:
This is merge conflicts, resolve
Link: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line